config/locales/en.yml

```diff
@@ -54,16 +53,41 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    signup_notification:
+      subject: "[%{app_name}] Confirmation" 
+      faq: "FAQ"
+      admin_posts: "Archive admin posts"
+      contact_support: "get in touch with our Support team"
+      activate_your_account: "activate your account"
+      welcome: "Welcome to the Archive of Our Own, %{login}!"
+      bye: "We hope you enjoy using the Archive."
+      text:
+        part1: "Please activate your account: %{activate_account_url}"
+        part2: |-
+                  Once your account is up and running, you can post your fanworks, set up email
+                  subscriptions to let you know when your favorite creators or works have updated,
+                  set preferences to customize the way the site looks and works for you, keep
+                  track of the works you've viewed on the Archive via your history, and much more.
+        part3: |-
+                  There's lots of information and advice on how to use the Archive in our FAQ
+                  at %{faq_url}. You'll
+                  find the latest news about site developments in our Archive admin posts at
+                  %{admin_posts_url}. If you need more
+                  help, run into a bug, or have questions or comments, please get in touch with
+                  our Support team, who are always happy to help out:
+                  %{contact_support_url} .
```

Comment:
The space before the period should probably be removed here, since `%{faq_url}.` and `%{admin_posts_url}.` don't have them.